### PR TITLE
[Docs] Fix typos, improve, update at Tutorials page

### PR DESCRIPTION
**Pull request by standardAI**
# What does this PR do?

Hi! Thank you for this awesome library!
I am new to open source. Firstly, I am reading the documentation. Secondly, I will read the codebase. Thirdly, I plan to make documentation and code contributions. I am open to your guidance.
I would be grateful for your feedback. @stevhliu @yiyixuxu @patrickvonplaten

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [X] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/en/tutorials/using_peft_for_inference.md [hunk 1]
```diff
@@ -150,16 +152,18 @@ image
 
 ## Monitoring active adapters
```
Comment (DN6) [inline docs/source/en/tutorials/using_peft_for_inference.md:153]: @sayakpaul Could you please check this section.

File: docs/source/en/tutorials/autopipeline.md [hunk 2]
```diff
@@ -34,13 +47,17 @@ image = pipeline(prompt, num_inference_steps=25).images[0]
 
 Under the hood, [`AutoPipelineForText2Image`]:
 
-1. automatically detects a `"stable-diffusion"` class from the [`model_index.json`](https://huggingface.co/runwayml/stable-diffusion-v1-5/blob/main/model_index.json) file
-2. loads the corresponding text-to-image [`StableDiffusionPipline`] based on the `"stable-diffusion"` class name
+1. Automatically detects a `"stable-diffusion"` class from the [`model_index.json`](https://huggingface.co/runwayml/stable-diffusion-v1-5/blob/main/model_index.json) file.
+2. Loads the corresponding text-to-image [`StableDiffusionPipeline`] based on the `"stable-diffusion"` class name.
```
Comment (stevhliu) [inline docs/source/en/tutorials/autopipeline.md:51]: No need to change anything here

File: docs/source/en/tutorials/using_peft_for_inference.md [hunk 3]
```diff
@@ -10,11 +10,13 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-[[open-in-colab]] 
+[[open-in-colab]]
+<!--Notebook links don't work.
+-->
```
Comment (stevhliu) [inline docs/source/en/tutorials/using_peft_for_inference.md:15]:
```suggestion
```

File: docs/source/en/tutorials/using_peft_for_inference.md [hunk 4]
```diff
@@ -150,16 +152,18 @@ image
...
 
-You have attached multiple adapters in this tutorial, and if you're feeling a bit lost on what adapters have been attached to the pipeline's components, you can easily check the list of active adapters using the [`~diffusers.loaders.get_active_adapters`] method:
+You have attached multiple adapters in this tutorial, and if you're feeling a bit lost on what adapters have been attached to the pipeline's components, you can easily check the list of active adapters using the [`~diffusers.loaders.LoraLoaderMixin.get_active_adapters`] method:
 
-```python
-active_adapters = pipe.get_active_adapters()
->>> ["toy", "pixel"]
+```py
+>>>active_adapters = pipe.get_active_adapters()
+>>>active_adapters
```
Comment (stevhliu) [inline docs/source/en/tutorials/using_peft_for_inference.md:159]:
```suggestion
active_adapters = pipe.get_active_adapters()
active_adapters
```

File: docs/source/en/tutorials/using_peft_for_inference.md [hunk 5]
```diff
@@ -150,16 +152,18 @@ image
...
 
-You can also get the active adapters of each pipeline component with [`~diffusers.loaders.get_list_adapters`]:
+You can also get the active adapters of each pipeline component with [`~diffusers.loaders.LoraLoaderMixin.get_list_adapters`]:
 
-```python
-list_adapters_component_wise = pipe.get_list_adapters()
->>> {"text_encoder": ["toy", "pixel"], "unet": ["toy", "pixel"], "text_encoder_2": ["toy", "pixel"]}
+```py
+>>>list_adapters_component_wise = pipe.get_list_adapters()
+>>>list_adapters_component_wise
```
Comment (stevhliu) [inline docs/source/en/tutorials/using_peft_for_inference.md:167]:
```suggestion
list_adapters_component_wise = pipe.get_list_adapters()
list_adapters_component_wise
```

**Review by stevhliu — APPROVED** [review]
Thanks! 🚀

File: docs/source/en/tutorials/autopipeline.md (hunk 2 above)
Comment (standardAI) [reply docs/source/en/tutorials/autopipeline.md:51]: Reversed except for a typo at StableDiffusionPipline.

File: docs/source/en/tutorials/using_peft_for_inference.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/tutorials/using_peft_for_inference.md:151]: Code checks out here so I'm going to merge 😄

**Pull request merged**